pkg/handlers/v1/cloud_fetch.go

```diff
@@ -38,6 +39,14 @@ type CloudAssetFetchByHostnameParameters struct {
 	Timestamp string `json:"time"`
 }
 
+// CloudAssetFetchAllByTimestampParameters represents the incoming payload for bulk fetching cloud assets for point in time with optional pagination
+type CloudAssetFetchAllByTimestampParameters struct {
+	Timestamp string `json:"time"`
+	// we use the pointer type to detect if the value was not present in input as otherwise the int variable would be 0, which is a valid input
```

Comment:
question: if 0 is invalid, why not just check to see if value == 0? Also, it looks like you are using 0 for the default value of offset. Is it a pointer here for consistency?

Reply:
The pointer use if for "generic" way to distinguish between the case when the value passed was zero vs. no value passed. I think I can change this to integer and just treat 0 as 'no value passed or 0 passed'. Good point, it makes little sense in this particular case.